From 88559f76c6983eee63023dba147d46fd645c4adb Mon Sep 17 00:00:00 2001
From: olwhistle <olwhistle@local>
Date: Thu, 17 Sep 2026 00:41:17 +0800
Subject: [PATCH 3/8] =?UTF-8?q?device+kernel:=20=E7=BB=91=E5=AE=9AdoorGuar?=
 =?UTF-8?q?d=20buildroot=E9=85=8D=E7=BD=AE;K7=20=E4=BD=BF=E8=83=BD5?=
 =?UTF-8?q?=E5=AF=B8MIPI=E5=B1=8F?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

- 设备defconfig 加 RK_BUILDROOT_BASE_CFG 指向 doorGuard
- rk3576-kickpi-k7-linux.dts include F050008M01 屏 dtsi(背光/面板/GT9xx触摸)
---
 .../.chips/rk3576/rockchip_rk3576_kickpi_k7_buildroot_defconfig  | 1 +
 .../arch/arm64/boot/dts/rockchip/rk3576-kickpi-k7-linux.dts      | 1 +
 2 files changed, 2 insertions(+)

diff --git a/device/rockchip/.chips/rk3576/rockchip_rk3576_kickpi_k7_buildroot_defconfig b/device/rockchip/.chips/rk3576/rockchip_rk3576_kickpi_k7_buildroot_defconfig
index cab94d84c..2307c5901 100755
--- a/device/rockchip/.chips/rk3576/rockchip_rk3576_kickpi_k7_buildroot_defconfig
+++ b/device/rockchip/.chips/rk3576/rockchip_rk3576_kickpi_k7_buildroot_defconfig
@@ -1,3 +1,4 @@
 RK_UBOOT_SPL=y
 RK_KERNEL_DTS_NAME="rk3576-kickpi-k7-linux"
 RK_USE_FIT_IMG=y
+RK_BUILDROOT_BASE_CFG="rk3576_kickpi_k7_doorGuard"
diff --git a/kernel-6.1/arch/arm64/boot/dts/rockchip/rk3576-kickpi-k7-linux.dts b/kernel-6.1/arch/arm64/boot/dts/rockchip/rk3576-kickpi-k7-linux.dts
index b3eeaace5..2a8dc28c6 100755
--- a/kernel-6.1/arch/arm64/boot/dts/rockchip/rk3576-kickpi-k7-linux.dts
+++ b/kernel-6.1/arch/arm64/boot/dts/rockchip/rk3576-kickpi-k7-linux.dts
@@ -8,6 +8,7 @@
 
 #include "rk3576.dtsi"
 #include "rk3576-kickpi-k7.dtsi"
+#include "rk3576-kickpi-lcd-mipi-5-720-1280-F050008M01.dtsi"
 #include "rk3576-linux.dtsi"
 
 / {
-- 
2.53.0

